sdk/program/src/syscalls/definitions.rs

```diff
@@ -44,7 +44,6 @@ define_syscall!(fn sol_create_program_address(seeds_addr: *const u8, seeds_len:
 define_syscall!(fn sol_try_find_program_address(seeds_addr: *const u8, seeds_len: u64, program_id_addr: *const u8, address_bytes_addr: *const u8, bump_seed_addr: *const u8) -> u64);
 define_syscall!(fn sol_sha256(vals: *const u8, val_len: u64, hash_result: *mut u8) -> u64);
 define_syscall!(fn sol_keccak256(vals: *const u8, val_len: u64, hash_result: *mut u8) -> u64);
-define_syscall!(fn sol_secp256k1_recover(hash: *const u8, recovery_id: u64, signature: *const u8, result: *mut u8) -> u64);
```

Comment:
Is this intended?

Reply:
Yes, I defined it manually in the secp256k1-recover crate to avoid depending on solana-program in that crate:
```
#[cfg(target_os = "solana")]
extern "C" {
    pub fn sol_secp256k1_recover(
        hash: *const u8,
        recovery_id: u64,
        signature: *const u8,
        result: *mut u8,
    ) -> u64;
}
```

Reply:
update: since #1827 was merged i just used the define_syscall macro instead of defining the syscall manually